frontend/src/app/_shared/shared.module.ts

```diff
@@ -12,6 +12,8 @@ import { SpacesToDashesPipe } from './../_pipes/spaces-to-dashes.pipe';
 import { TrackScrollDirective } from './../_directives/scroll.directive';
 import { InViewportModule } from 'ng-in-viewport';
 import { Ng2AutoCompleteModule } from 'ng2-auto-complete';
+import { LineBreakFormatterPipe } from '../_pipes/line-break-formatter.pipe';
+import { ChunkArrayPipe } from '../_pipes/chunk-array.pipe';
```

Comment:
Again, if this is only currently used in one place, does it need to be in the shared module?

Reply:
@lauraGgit moving this to trees.module.ts